hack/jenkins/e2e.sh

```diff
@@ -116,40 +117,24 @@ GCE_SLOW_TESTS=(
 
 # Tests which are not able to be run in parallel.
 GCE_PARALLEL_SKIP_TESTS=(
-    "Etcd"
-    "NetworkingNew"
-    "Nodes\sNetwork"
-    "Nodes\sResize"
-    "MaxPods"
-    "ResourceUsage"
-    "SchedulerPredicates"
-    "Services.*restarting"
-    "Shell.*services"
+    "Etcd"                  # simulate Etcd failures.
+    "Nodes\sNetwork"        # simulate node network failures.
+    "Nodes\sResize"         # add and remove nodes to and from the cluster
+    "MaxPods"               # saturates nodes with pods
+    "ResourceUsage"         # TODO: Not clear why this is here?
+    "SchedulerPredicates"   # TODO: Perhaps enable some of these?
+    "restarting\skube-proxy"# breaks pod and service networking
     )
 
 # Tests which are known to be flaky when run in parallel.
 GCE_PARALLEL_FLAKY_TESTS=(
-    "DaemonRestart"
-    "Elasticsearch"
-    "PD"
-    "ServiceAccounts"
-    "Services.*change\sthe\stype"
-    "Services.*functioning\sexternal\sload\sbalancer"
-    "Services.*identically\snamed"
     "Services.*release.*load\sbalancer"
-    "Services.*endpoint"
```

Comment:
on most recent PR Jenkins run, "Service endpoints latency should not be very high" failed. Seems it's still flaky?